handlers/strike.py

```diff
@@ -0,0 +1,43 @@
+import os
+from jsonpath_rw import jsonpath, parse
+
+class Strike():
+	def handleCommand(self, command):
+	
+		if command.command == "getStrikes"
+			return getStrikesString(self)
+		userId = parse('$.message.from.id').find(command.parent)
+		strikeUser = command.args
+		if not userId or not strikeUser:
+			return None
+		userId = userId[0].value
+		userFile = 'strike/{0}'.format(strikeUser)
+		if not userId == 313082320 #Ryan's id
+			return "Pshhh. You cannot assign strikes."
+		if not os.path.isfile(userFile):
+		  with open(userFile, "w") as f:
+			f.write("{0}\n1".format(strikeUser))
+		else:
+		  with open(userFile, "r+") as f:
+			data = f.read().split()
+			nStrikes = int(data[1]) + 1
+			f.seek(0)
+			f.write("{0}\n{1}".format(strikeUser, nStrikes))
+			f.truncate()
+		return "Careful {0}. You are now at {1} strikes\n".format(strikeUser, nStrikes)
```

Comment:
don't need the \n at end of line (i think that will just make the message bigger in telegram?)

Reply:
Done